client/dashboard/data/index.ts

```diff
@@ -27,19 +24,15 @@ export const updateProfile = async ( data: Partial< Profile > ) => {
 			delete data[ key as keyof Profile ];
 		}
 	}
-	return await wpcom.req.post( {
-		path: '/me/settings?http_envelope=1',
```

Comment:
Thank you, glad to see half of the manual needless `http_envelope` query arg instances removed 👏